packages/dappmanager/src/modules/ethClient/changeEthMultiClient.ts

```diff
@@ -43,3 +61,80 @@ export async function changeEthMultiClient(
     eventBus.runEthClientInstaller.emit();
   }
 }
+
+// Utils
+
+export async function migrateEthClientFullNode(
+  dnpName: string,
+  remove: boolean
+): Promise<void> {
+  const previousEthClientPackage = await packageGet({
+    dnpName
+  });
+
+  // Eth clients should have just one service
+  if (previousEthClientPackage.containers.length > 1)
+    throw Error("Unexpected number of services in ETH client 1");
```

Comment:
This error can be very dangerous, having an Eth1 node with two containers is something that can happen in the future. This error will break all functionality.